frameworks/digital-outcomes-and-specialists/questions/briefs/questionAndAnswerSessionDetailsSpecialists.yml

```diff
@@ -0,0 +1,46 @@
+id: questionAndAnswerSessionDetails
+name: Question and answer session details
+question: Question and answer session details
+question_advice: |
+  <br>
```

Comment:
Yeah, that's a good question. Probably not...but worth checking with @ralph-hawkins